app/assets/stylesheets/static_pages.scss

```
@@ -496,3 +497,58 @@ table a, #comments a{background-color:inherit;}
   .font-x2{font-size:1.4rem;}
   .font-x3{font-size:1.6rem;}
 }
+
+
+.product-box h3>a,.product-box h3>a:hover,.product-box h3>a:focus .product-box a{
+  color: black;
```

```suggestion
  color: $black;
```
Xong ở đây thì gọi thế này

---

Dạ